website/docs/content/writing-style/index.md

```diff
@@ -5,8 +5,6 @@ description: Best practices for writing style, grammar, and formatting.
 previewImage: assets/illustrations/content/writing-style.jpg
 ---
 
-Generally, HashiCorp follows the [2020 AP Stylebook](https://www.apstylebook.com/ap_stylebook) rules and guidelines, unless HashiCorp specific style is indicated.
-
```

Comment:
Do we still follow AP Style generally though? I think this is useful information to know if so. I would just remove the link in that case.

Reply:
The writer.com resource is great! Thanks for sharing @KristinLBradley. 

@andgen404 Can you update the text to use Kristin's second example and add the link to the writer.com resource? This feels more helpful for consumers 🙂